lib/travis/api/app/endpoint/pusher.rb

```diff
@@ -5,27 +5,27 @@ class Endpoint
     class Pusher < Endpoint
       PRIVATE_CHANNEL_FORMAT = /private-(?<type>\w+)-(?<id>\d+)\.?\w*/
 
-      post '/auth', scope: :private do
+      post '/auth' do
         { channels: Hash[*authorized_channels.compact.flatten] }
       end
 
       private
 
         def authorized_channels
           params[:channels].map do |channel|
-            [channel, authenticate(channel)] if signed_in? && authorized_for?(channel)
+            [channel, authenticate(channel)] if authorized_for?(channel)
           end
         end
 
         def authorized_for?(channel)
-          return false unless matches = PRIVATE_CHANNEL_FORMAT.match(channel)
+          return true unless matches = PRIVATE_CHANNEL_FORMAT.match(channel)
 
           id = matches[:id].to_i
           case matches[:type]
           when 'user'
-            current_user.id == id
+            signed_in? && current_user.id == id
           when 'repo'
```

Comment:
do we use `repo` for Build status updates?

Reply:
i believe so, but i'm not sure :)

Reply:
so then you wouldn't get these updates unless you are signed in?

Reply:
i don't think we're using `repo` channels any more anywhere, but i'd rather not remove it right now